src/redux/books/books.js

```diff
@@ -0,0 +1,55 @@
+import { v4 } from 'uuid';
+
+const ADD = 'bookstore/books/ADD';
+const REMOVE = 'bookstore/books/REMOVE';
+
+// Actions
+const addBook = (book) => ({
+  type: ADD,
+  book,
+});
+
+const removeBook = (id) => ({
+  type: REMOVE,
+  id,
+});
+
+// Reducer
+const bookReducer = (state = [
+  {
+    id: v4(),
+    title: 'The Pearl of a great price',
+    author: 'Joseph Smith',
```

Comment:
- [ ] As per the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md), the `initial state` is to be an empty array of books. Currently, your array is already populated with static data. :+1: 
- It is important to include an empty books array in the initialState because it defines the initial state of the books data in the Redux store. When the application starts, the Redux store is initialized with the `initialState`, which includes the initial value of the books state variable. By setting an empty array as the initial value, we ensure that the books state variable is defined, even if there are no books in the store yet. 
- This makes it easier to work with the books state variable throughout the application, as we can assume that it always exists and is an array. So kindly implement this :wink: 

```
const initialState = {
  books: [],
}
```